apps/modernization-ui/src/SkipLink/SkipLink.scss

```diff
@@ -0,0 +1,6 @@
+.hidden-link {
```

Comment:
suggestion: should the class name match the Component name to keep things consistent?